app/controllers/api/v1/trips_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::TripsController < ApplicationController
+
+  def create
+    trip = Trip.create(name: params["destination"],
```

Comment:
so, if it is a `destination` why is the attribute named `name` on the model?

Reply:
Changed to destination with the nicer ring to it.